tests/server/server-executeCommand-002.phpt

```diff
@@ -0,0 +1,78 @@
+--TEST--
+MongoDB\Driver\Server::executeCommand() takes a read preference
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; NEEDS("REPLICASET"); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = new MongoDB\Driver\Manager(REPLICASET);
+
+$rp = new MongoDB\Driver\ReadPreference(MongoDB\Driver\ReadPreference::RP_SECONDARY);
+$secondary = $manager->selectServer($rp);
+
+$command = new MongoDB\Driver\Command(array('profile' => 2));
+$cursor = $secondary->executeCommand(DATABASE_NAME, $command);
+$result = current($cursor->toArray());
+
+if (empty($result->ok)) {
+    exit("Could not set profile level\n");
```

Comment:
I just wanted to abort the test at this point. Throwing an exception would also work, but this seemed simpler.